test/integration/middleware/with-i18n/next.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  i18n: {
+    locales: ['default', 'en', 'es', 'fr', 'zh'],
+    defaultLocale: 'default',
+    localeDetection: false,
```

Comment:
Why is `localeDetection: false` here? The default is enabled so seems like we should test against the default

Reply:
I removed it. I was reproducing the issue from a report but forgot to remove the option back, there is no point on keeping it indeed. Thanks!